src/main/java/database/DBService.java

```diff
@@ -0,0 +1,52 @@
+package database;
+
+import dto.ChessGameDto;
+import java.sql.Connection;
+import model.Camp;
+import model.piece.Piece;
+import model.position.Position;
+
+public class DBService {
+
+    private final ChessBoardDao chessBoardDao;
+    private final ChessGameDao chessGameDao;
+
+    public DBService(Connection connection) {
+        this.chessBoardDao = new ChessBoardDao(connection);
+        this.chessGameDao = new ChessGameDao(connection);
+    }
+
+    public ChessGameDto reload() {
+        return new ChessGameDto(chessBoardDao.findAll(), chessGameDao.find());
+    }
+
+    public boolean isContinue() {
+        return chessGameDao.find() != null;
```

Comment:
```suggestion
        return chessGameDao.find().describeConstable().isEmpty();
```

자바에서 제공하는 메서드를 활용해볼까요?